index.html

```diff
@@ -70,14 +79,66 @@ <h3>Tonic</h3>
 
                 </li>
                 <li class="view">
-                    <a href="" class="buttons">
-                        <h3>See Project</h3>
+                    <a href="#firstBtn" class="buttons">
+                        <h3 id="firstBtn">See Project</h3>
                     </a>
                 </li>
             </ul>
         </div>
-        <div class="projects reverse">
-            <img class="img" src="./images/Snapshoot Portfolio 2.png" alt="Project Image">
+        <!-- POPUP WINDOW -->
+        <section class="popup" id="popup">
+            <ul class="popup-image">
+                <li class='popup-header'>
+                    <h2 id="popup-header">Tonic</h2>
+                    <a href="#">
+                        <h4 id="cancel-popup">X</h4>
+                    </a>
+                </li>
+                <li class="popup-stack">
+                    <ul>
+                        <li class="canopy" id="popupstack1">CANOPY</li>
+                        <li class="bed" id="popupstack2">Back End Dev</li>
+                        <li class="year" id="popupstack3">2015</li>
+                    </ul>
+                </li>
+                <li class="popup-img" id="popup-img">
+
+                </li>
+            </ul>
+            <ul class="popup-info">
+                <li class="popupParagraph">
+                    <p id="popup-lorem">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+                        Ipsum has been
+                        the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley
+                        of type and scrambled it to make a type specimen book. It has survived not only five centuries,
+                        but also the leap into electronic typesetting, remaining essent</p>
+                </li>
+                <li class="popuplang lang">
+                    <ul class="popuplang-ul">
+                        <li class="html">html</li>
+                        <li class="ruby">Ruby</li>
+                        <li class="css">css</li>
+                        <li class="js">javaScript</li>
+                    </ul>
+                    <ul>
+                        <li class="popupBtn">
+                            <div>
+                                <a href="#" class="buttons">See live<img src="./images/Icon.png" alt="github icon"></a>
+                            </div>
+                            <div> <a href="#" class="buttons">See Source<img src="./images/Vector22.png"
+                                        alt="github icon"></a>
+                            </div>
```

Comment:
- [ ] Hi @David-Lanzz, well done working on the pop-up section of your project, 👍 👍 but according to the project requirement, it is paramount that the pop-up is created dynamically and the hardcoded should be removed from the HTML file. I would kindly suggest you create the pop-up dynamically and remove the code you have in the HTML file to see if the dynamic function works fine in your JavaScript file 👍 

  - N.B: The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.** Check an example of a student's projects: [Link 1](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [Link 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96)